pom.xml

```diff
@@ -53,28 +67,23 @@
     <dependency>
       <groupId>io.vertx</groupId>
       <artifactId>vertx-core</artifactId>
-      <version>${vertx.version}</version>
```

Comment:
How does maven know what version to use with these version's removed?

Reply:
I guess version is inherited from the [dependency management section](https://github.com/folio-org/mod-circulation/pull/580/files#diff-600376dffeb79835ede4a0b285078036R40-R46)

Reply:
Yes.

Reply:
Thanks. I don't really know what that does. Could either of you give me a simple overview?

Is that a special kind of dependency that defines the intended versions of a variety of potential dependencies, some of which may be later included as actual dependencies?

Reply:
https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Management